.github/CODEOWNERS

```diff
@@ -56,6 +52,29 @@
 /Code/Tools/SceneAPI/ @o3de/sig-content-maintainers
 /Gems/SceneProcessing/ @o3de/sig-content-maintainers
 
+# SIG-Content Viewport ownership area
+/Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/FocusMode/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/Picking/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ @o3de/sig-content-maintainers
+/Code/Framework/AzFramework/AzFramework/Viewport/ @o3de/sig-content-maintainers
+/Code/Framework/AzFramework/AzFramework/Visibility/ @o3de/sig-content-maintainers
+
+# SIG-Content White Box ownership area
+/Gems/WhiteBox/ @o3de/sig-content-maintainers
```

Comment:
Missing the whitebox automated test folder:

````
/AutomatedTesting/Gem/PythonTests/WhiteBox
````

Reply:
Ah yes well remembered, give me 5 mins 👍

Reply:
Updated 👍